addons/supplies/XEH_postInitServer.sqf

```diff
@@ -0,0 +1,15 @@
+#include "script_component.hpp"
+
+playerTraits = createHashMap;
+
+[QGVAR(storeTraits), {
+    playerTraits set _this;
+}] call CBA_fnc_addEventHandler;
+
+[QGVAR(resetTraits), {
+    private _type = playerTraits getOrDefault [getPlayerUID _this, ""];
+    if (typeOf _this isEqualTo "tacs_Unit_B_TeamLeader") exitWith {};
```

Comment:
Do we really need this? Hardcoding that class seems redundant, TLs shouldn't take those traits anyways, and if they do, @Lairdd1989 will hunt them down.

Reply:
Was requested by @TyroneMF

Reply:
Then I would do it like this and define the array (lower-cased!) in `preInit` so it can be changed at runtime (or even a CBA Setting).
```suggestion
    if (toLower (typeOf _this) in GVAR(traitsBlacklist)) exitWith {};
```